### PR TITLE
document reward-minting taylorSeriesExpansion

### PR DESCRIPTION
**Pull request by davidad**
Hoping to pay down my share of the technical debt mentioned in #317, by cleaning up and adding many paragraphs of documentation to the algorithm I devised for computing f(t)=1-(1/2)^(t/6yr) to atto-level precision. Please let me know if there are further questions about what's going on here.

### CODE REVIEW

File: actors/builtin/reward/reward_state.go [hunk 1]
```diff
@@ -49,31 +40,174 @@ func ConstructState(emptyMultiMapCid cid.Cid) *State {
...
+//
+//   When we have the numerator and denominator for a given term set up, we
+//   compute their fixed-point fraction by left-shifting the numerator before
+//   performing integer division.
+//
+//   Finally, at the end of each loop, we remove unnecessary bits of precision
+//   from both the numerator and denominator accumulators to control the
+//   computational complexity of the bigint multiplications.
+
+// Fixed-point precision used internally and for output
```
Comment (anorth) [inline actors/builtin/reward/reward_state.go:134]:
```suggestion
// Fixed-point precision (in bits) used internally and for output
```

File: actors/builtin/reward/reward_state.go [hunk 2]
```diff
@@ -49,31 +40,174 @@ func ConstructState(emptyMultiMapCid cid.Cid) *State {
...
+const FixedPoint = 97
+
+// Used in the definition of λ
+const BlockTimeSeconds = 30
+const SecondsInYear = 31556925
+
+// The following are the numerator and denominator of -ln(1/2), represented as
```
Comment (anorth) [inline actors/builtin/reward/reward_state.go:141]: OMG that helps so much, thank you. Could we name it something like `NegLogHalf` too?

File: actors/builtin/reward/reward_state.go [hunk 3]
```diff
@@ -49,31 +40,174 @@ func ConstructState(emptyMultiMapCid cid.Cid) *State {
...
+//
+//   Finally, at the end of each loop, we remove unnecessary bits of precision
+//   from both the numerator and denominator accumulators to control the
+//   computational complexity of the bigint multiplications.
+
+// Fixed-point precision used internally and for output
+const FixedPoint = 97
+
+// Used in the definition of λ
+const BlockTimeSeconds = 30
```
Comment (anorth) [inline actors/builtin/reward/reward_state.go:138]: FYI this number is wrong, and has a different value elsewhere in this repo. But that's tracked as a thing to fix (with a structural fix, not just changing the literal here)

Comment (davidad) [reply actors/builtin/reward/reward_state.go:138]: Yeah, I saw that mentioned in #317, and did some grepping around, but I was unable to find the other value.

File: actors/builtin/reward/reward_state.go (hunk 2 above)
Comment (davidad) [reply actors/builtin/reward/reward_state.go:141]: Sure!

Comment (davidad) [reply actors/builtin/reward/reward_state.go:141]: Sure!

**Pull request merged**